spawnpool.py

```diff
@@ -205,25 +204,29 @@ def summarize(message, list):
             self._heart_beating = False
 
     @gen.coroutine
-    def _launch_container(self, path=None, enpool=True):
+    def _launch_container(self, user=None, enpool=True):
         '''Launch a new notebook server in a fresh container, register it with the proxy, and
         add it to the pool.'''
 
-        if path is None:
-            path = user_prefix()
+        if user is None:
+            user = new_user()
 
-        # This must match self.name_pattern or Bad Things will happen.
+        path = "user/" + user
+
+        # This must match self.container_name_pattern or Bad Things will happen.
```

Comment:
Yours. I just updated the comment to reflect `container_name_pattern` instead of `name_pattern`.